core/templates/dev/head/pages/exploration_editor/statistics_tab/MultipleIncorrectIssueDirective.js

```diff
@@ -0,0 +1,162 @@
+                },
+                controller: [
+                  '$scope', '$uibModalInstance', 'playthroughIndex',
+                  'playthrough', 'AlertsService', 'LearnerActionRenderService',
+                  function(
+                      $scope, $uibModalInstance, playthroughIndex,
+                      playthrough, AlertsService, LearnerActionRenderService) {
+                    $scope.playthroughIndex = playthroughIndex;
+
+                    $scope.displayBlocks =
+                      LearnerActionRenderService.getDisplayBlocks(
+                        playthrough.actions);
+
+                    var blockActionIndexMapping = {};
+                    var i = 0;
+                    var total = $scope.displayBlocks.reduce(
+                      function(runningTotal, displayBlock) {
+                        blockActionIndexMapping[i] = runningTotal;
+                        i += 1;
+                        return runningTotal + displayBlock.length;
+                      }, 1);
+
+                    $scope.maxHidden = $scope.displayBlocks.length - 1;
+
+                    $scope.getDisplayBlockIndex = function(displayBlock) {
+                      return $scope.displayBlocks.indexOf(displayBlock);
+                    };
+
+                    $scope.isDisplayBlockOnInitDisplay = function(block) {
+                      return $scope.getDisplayBlockIndex(block) === 0;
+                    };
+
+                    $scope.createDisplayBlockNavId = function(block) {
+                      return $scope.getDisplayBlockIndex(block) + 1;
+                    };
+
+                    $scope.renderBlockHtml = function(displayBlock) {
+                      var index = $scope.getDisplayBlockIndex(displayBlock);
+                      if (index == 0) {
+                        var service = LearnerActionRenderService
+                        return service.renderFinalDisplayBlockForMISIssueHTML(
+                          displayBlock, blockActionIndexMapping[index]);
+                      }
+                      return LearnerActionRenderService.renderDisplayBlockHTML(
+                        displayBlock, blockActionIndexMapping[index]);
+                    };
+
+                    $scope.showRemainingActions = function(pIdx) {
+                      if ($scope.maxHidden === 1) {
+                        document.getElementById(
+                          'remainingActions' + pIdx.toString() +
+                          $scope.maxHidden.toString()).style.display = 'block';
+                        document.getElementById('arrowDiv').style.display =
+                          'none';
+                      } else {
+                        var currentShown = 0, i;
+                        for (i = $scope.maxHidden; i > 0; i--) {
+                          if (document.getElementById(
+                            'remainingActions' + pIdx.toString() +
+                            i.toString()).style.display === 'block') {
```

Comment:
Let's just keep track of the one currently shown using a variable?

Reply:
@brianrodri This will be repeated through three PRs, ive added a log on my dev log to change this in all three PRs at one go.